### ISSUE TITLE
Hiding actions using display:none causes keyboard a11y issues

### ISSUE THREAD

**Issue by alexandrudima** [Microsoft/vscode#2329]
Testing #2145

In the working files:
* tab until the last action
* tab takes you to the file
* tab takes you to the close file action <--- missed by shift-tab
* tab takes you to the file/folder explorer
> Shift-tab should take you back to the close file action, instead it jumps straight to the file

**Comment by alexandrudima** [comment]
The same is true for the actions in the file/folder explorer. The actions are only reachable via "Tab". They should be reachable via Shift-Tab as well

**Comment by alexandrudima** [comment]
I think this is a problem with all "hidden" actions ?

**Comment by bpasero** [comment]
We should not use "display: none" but some other CSS property to hide actions (e.g. width). I implemented this for split view actions, but not yet for trees.
